src/uncontrast/pages/UploadNewPaper/ExperimentsSection/SuppressedStimuli.jsx

```diff
@@ -362,9 +360,14 @@ export default function SuppressedStimuli({
                           }
                         />
                         <CustomSelect
-                          noBlank
                           disabled={disableCondition}
-                          value={fieldValue.is_target_stimulus}
+                          value={
```

Comment:
This is very hard to understand, can you make it simpler or extract to a function